pkg/deploy/generator/scripts/rpVMSS.sh

```diff
@@ -169,6 +169,18 @@ cat >/etc/fluentbit/fluentbit.conf <<'EOF'
 	Match journald
 	Rule $LOGKIND ifxaudit ifxaudit false
 
+[FILTER]
+	Name rewrite_tag
+	Match journald
+	Rule $LOGKIND outboundRequests outboundRequests false
+
+[FILTER]
+	Name modify
+	Match  outboundRequests
+	Remove CLIENT_PRINCIPAL_NAME
+	Remove FILE
+	Remove COMPONENT
+
```

Comment:
FYI: Fluentbit config to log to a different Geneva table (explained in the Splike)